src/pe_reports/report_generator.py

```diff
@@ -55,6 +62,16 @@ def load_template():
     return prs
 
 
+def load_customers():
+    """Export PowerPoint report set to output directory."""
+    try:
+        with open(CUSTOMERS) as customers_file:
+            names_obj = json.load(customers_file)
+    except FileNotFoundError as not_found:
+        logging.error("%s : Missing input data. No report generated.", not_found)
+    return names_obj
```

Comment:
This should be initialized to an empty dictionary before the `try` block to cleanly handle the peculiar case of the file missing.

Reply:
@mcdonnnj , 

Implemented recommendation per commit [a29600a](https://github.com/cisagov/pe-reports/pull/33/commits/a29600a58b1bc97062263a84399fa18a45a52cd8)